ajax/libs/jsignature/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jsignature",
+  "version": "2.1.2",
+  "filename": "jSignature.min.js",
+  "description": "jQuery plugin for handling digital signatures",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brinley/jSignature"
+  },
+  "keywords": [
+    "jSignature",
+    "signature",
+    "jQuery"
+  ],
+  "author": "Brinley Ang",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brinley/jSignature.git",
+    "fileMap": [
+      {
+        "basePath": "libs",
+        "files": [
+          "**/*"
```

Comment:
Have a[ look here](https://github.com/brinley/jSignature/tree/v2.1.2#adding-jsignature-to-your-page) which files were used bu author and update the pattern accordingly